docs/src/modules/sandbox/CodeSandbox.ts

```diff
@@ -39,7 +39,7 @@ const createReactApp = (demo: {
   raw: string;
   codeVariant: CodeVariant;
   githubLocation: string;
-  product?: Product;
+  productId?: Product;
```

Comment:
We could almost change the type import, to use `MuiProductId` from `getProductInfoFromUrl`